src/app/CommandInputField.tsx

```diff
@@ -14,6 +12,9 @@ import { action } from "mobx";
 export const CommandInputField = observer((props: { controller: BottomPanelController }) => {
   const { app, logger } = getRootStore();
   const controller = props.controller;
+  let lastCmdIndex = 0;
```

Comment:
useState or Mobx observer should be used instead

Reply:
Updated, it changed to observable now. Thank you.